### PR TITLE
Issue-36 

### LINKED ISSUE #36: .gitignore redundant in modules
The modules (well, actually the copied modules files) do not need to contain a .gitignore file because there is already one in the `neubot` directory.

### PR DESCRIPTION
**Pull request by DavideAllavena**
Updated update_deps.sh to new commits without .gitignore.

Closes #36

### CODE REVIEW

File: scripts/update_deps.sh [hunk 1]
```diff
@@ -9,11 +9,11 @@ get() {
 }
 
 rm -rf tmp/*
-get DavideAllavena/neubot-mod-speedtest 1a55737 mod_speedtest
+get DavideAllavena/neubot-mod-speedtest 339e1e8 mod_speedtest
 get DavideAllavena/neubot-mod-bittorrent cda686c mod_bittorrent
```
Comment (bassosimone) [inline scripts/update_deps.sh]: You should also remove `.gitignore` for `mod_bittorrent`.

File: scripts/update_deps.sh [hunk 2]
```diff
@@ -9,11 +9,11 @@ get() {
...
-get DavideAllavena/neubot-mod-dash 05328c0 mod_dash
+get DavideAllavena/neubot-mod-dash d556977 mod_dash
 get DavideAllavena/neubot-lib-net beaafd7 lib_net
```
Comment (bassosimone) [inline scripts/update_deps.sh]: You should also remove `.gitignore` for `lib_net`.

File: scripts/update_deps.sh (hunk 1 above)
Comment (DavideAllavena) [inline scripts/update_deps.sh]: removed

File: scripts/update_deps.sh (hunk 2 above)
Comment (DavideAllavena) [inline scripts/update_deps.sh]: removed.

**Comment by bassosimone** [comment]
Hmm, I just cloned the repository, checked out to the right branch, executed the sync-up script.

Afterwards, this is what I've got:

```
$ git status
On branch feature/issue-36
Your branch is up-to-date with 'origin/feature/issue-36'.
Untracked files:
  (use "git add <file>..." to include in what will be committed)

    neubot/lib_net/.gitignore
    neubot/mod_bittorrent/.gitignore
    tmp/

nothing added to commit but untracked files present (use "git add" to track)
```

Perhaps you should also update the commit ref into the update script?

Thanks!

**Comment by DavideAllavena** [comment]
Sorry! File updated.

**Comment by bassosimone** [comment]
Excellent, thanks: I will now rebase, fixup and merge!

**Pull request merged**